### PR TITLE
Typo and indentation

### PR DESCRIPTION
**Pull request by s-en-o**
### Why:

Closes [issue link]

### What's being changed:
Typo and indentation

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Review by ramyaparimi — APPROVED** [review]
@s-en-o 
Thanks so much for opening a PR with this fix! I'll update the branch and get this merged once tests are passing 🍏

**Comment by ramyaparimi** [comment]
@s-en-o Can you enable the checkbox to allow maintainer edits? Then I'll be able to hit the update button and get this merged in. Thank you! https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by s-en-o** [comment]
Hi @ramyaparimi, it's always been enable the 'Allow edit by maintainers', unless I miss something?

File: content/github/authenticating-to-github/troubleshooting-ssh/using-ssh-over-the-https-port.md [hunk 1]
```diff
@@ -28,15 +28,15 @@ If that worked, great! If not, you may need to [follow our troubleshooting guide
 
 ## Enabling SSH connections over HTTPS
 
-If you are able to SSH into `git@ssh.{% data variables.command_line.backticks %}` over port 443, you can override your SSH settings to force any connection to {% data variables.product.product_location %} to run though that server and port.
```
Comment (MaryJaneLove4) [inline content/github/authenticating-to-github/troubleshooting-ssh/using-ssh-over-the-https-port.md:31]: Correct: Through

**Review by MaryJaneLove4 — APPROVED** [review]

**Review by lucalves — APPROVED** [review]

**Comment by ramyaparimi** [comment]
@s-en-o Could you please double check? I do not see the button to update the branch 🐢

**Comment by s-en-o** [comment]
> @s-en-o Could you please double check? I do not see the button to update the branch 🐢

@ramyaparimi 
image

Always enable.

**Comment by ramyaparimi** [comment]
@s-en-o Thanks!! I can see the button now !! Merging this PR 🍏

**Comment by s-en-o** [comment]
Thanks @ramyaparimi!

**Pull request merged**